app/services/code_base_full_text_search_service.go

```diff
@@ -0,0 +1,22 @@
+package services
+
+import (
+	"ai-developer/app/repositories"
+	"context"
+)
+
+type CodeBaseSearchService struct {
+	codeFullTextSearchRepository *repositories.CodeBaseOpenSearchRepository
```

Comment:
This has to be CodeBaseSearchRepository

Reply:
was using that only initially if you see my previous commit as we had to move away from 
`// Provide OpenSearch Repository
	err = c.Provide(func(client *opensearch.Client, logger *zap.Logger) repository.CodeBaseSearchRepository {`
	
	
	this was providing CodeBaseSearchRepository this as a particular implementation

Reply:
https://github.com/TransformerOptimus/SuperCoder/pull/76#discussion_r1680820194 Referring this